### PR TITLE
Added cronjob for focus-ios export_strings.yml

### PR DESCRIPTION
**Pull request by bcolsson**

### CODE REVIEW

File: .github/workflows/export_strings.yml [hunk 1]
```diff
@@ -1,5 +1,12 @@
 name: Import strings
 on:
+  schedule:
+    - cron: '0 22 * * 3' # At 15:00 PDT on Wednesday
```
Comment (flodolo) [inline .github/workflows/export_strings.yml:4]: I'd move this earlier, not sure why FxA picked such a weird time.

9 PDT would work, as there's a chance I will see it before my day ends, and you'll have the entire day to look at it.
```suggestion
    - cron: '0 16 * * 3' # At 9:00 PDT on Wednesday
```

**Review by flodolo — APPROVED** [review]

File: .github/workflows/export_strings.yml [hunk 2]
```diff
@@ -1,5 +1,12 @@
 name: Import strings
 on:
+  schedule:
+    - cron: '0 16 * * 3' # At 9:00 PDT on Wednesday
+    inputs:
```
Comment (flodolo) [inline .github/workflows/export_strings.yml:5]: I completely missed this (schedule doesn't support inputs). Fixed directly on main.

Comment (bcolsson) [reply .github/workflows/export_strings.yml:5]: Oh, that seems like a strange design constraint, but good to know. I see that the script is being run with `type ""` argument and executing successfully, so guessing if an empty value is provided the script resolves to standard?

Comment (flodolo) [reply .github/workflows/export_strings.yml:5]: Well, you can't pass a value to the cronjob at run-time, so it makes sense. What was very confusing is that the error pointed at a wrong format on line 3 (cron).

And yes, the script [has a default value of standard](https://github.com/mozilla-l10n/focusios-l10n/blob/main/.github/scripts/update_other_locales.py#L57).

Comment (bcolsson) [reply .github/workflows/export_strings.yml:5]: Ah, thanks for this. Helps refine my mental model of github actions.

**Pull request merged**